poly/constants.go

```diff
@@ -0,0 +1,15 @@
+package poly
+
+import (
+	"github.com/dedis/crypto/edwards"
+)
+
+const (
```

Comment:
This SECURITY variable address the concern we talked about the promise.go code. In the current version, a State has to have received all Responses before revealing any share. In this case, by setting SECURITY to MINIMUM, the State does NOT check if it has received all responses before revealing the share. THis behavior is needed in order to easily establish a shared secret as done in joint.go. Moderate is just extra, and can be removed without further notice.